tests/compilation/test_predictor_rl.py

```diff
@@ -24,13 +24,13 @@ def test_predictor_env_reset_from_string() -> None:
 
 @pytest.mark.parametrize(
     "figure_of_merit",
-    ["expected_fidelity", "critical_depth"],
+    ["expected_fidelity", "critical_depth", "expected_success_probability"],
```

Comment:
Can we directly point to the list of supported figures of merit in mqt.predictor.reward?